shared/message/motion/KickCommand.proto

```diff
@@ -32,8 +32,10 @@ enum KickCommandType {
 }
 
 message KickCommand {
-    vec3            target            = 1;  // The point to kick
-    vec3            direction         = 2;  // force is the magnitude
+    /// The point to kick
+    vec3 target = 1;
+    // Force is the magnitude
```

Comment:
Is this left over from an old force field?

Reply:
wdym?

Reply:
Oh does it mean the force of the kick is the magnitude of the direction vector? It didn't sound like it went with the field.

Reply:
Yea, it's meant to go with the field. I'll make it clearer.